UPGRADE-API-2.0.md

```diff
@@ -225,3 +189,97 @@ All the `setter` methods have been removed from the commands above and also ther
     - `sylius_api.serialization_groups.skip_adding_read_group`
     - `sylius_api.serialization_groups.skip_adding_index_and_show_groups`
+
+## Resource configuration changes
+
+### Updated API Routes
+
+* AvatarImage
+
+- `'GET' - /api/v2/admin/avatar-images/{id}` → `'GET' - /api/v2/admin/administrators/{id}/avatar-image`
+- `'POST' - /api/v2/admin/avatar-images` → `'POST' - /api/v2/admin/administrators/{id}/avatar-image`
+- `'DELETE' - /api/v2/admin/avatar-images/{id}` → `'DELETE' - /api/v2/admin/administrators/{id}/avatar-image`
+
+* ShopUser, CustomerPassword, and CustomerVerification
+
+- `'POST' - /api/v2/shop/reset-password-requests` → `'POST' - /api/v2/shop/reset-password`
+- `'PATCH' - /api/v2/shop/reset-password-requests/{resetPasswordToken}` → `'PATCH' - /api/v2/shop/reset-password/{resetPasswordToken}`
+- `'POST' - /api/v2/shop/account-verification-requests` → `'POST' - /api/v2/shop/verify-shop-user`
+- `'PATCH' - /api/v2/shop/account-verification-requests/{token}` → `'PATCH' - /api/v2/shop/verify-shop-user/{token}`
+
+* AdminUserPassword
+
+- `'POST' - /api/v2/admin/reset-password-requests` → `'POST' - /api/v2/admin/reset-password`
+- `'PATCH' - /api/v2/admin/reset-password-requests/{resetPasswordToken}` → `'PATCH' - /api/v2/admin/reset-password/{resetPasswordToken}`
+
+* CatalogPromotion
+
+- `'GET' - /api/v2/admin/catalog-promotion-actions/{id}` → `'GET' - /api/v2/admin/catalog-promotions/{code}/actions/{id}`
+- `'GET' - /api/v2/admin/catalog-promotion-scopes/{id}` → `'GET' - /api/v2/admin/catalog-promotions/{code}/scopes/{id}`
+
+* Translation
+
+- `'GET' - /api/v2/admin/[resource]-translations/{id}` → `'GET' - /api/v2/admin/[resource]/{id}/translations/{localeCode}`
+- `'GET' - /api/v2/admin/[resource]-translations/{code}` → `'GET' - /api/v2/admin/[resource]/{code}/translations/{localeCode}`
+
+* Channel Pricing
+
+- `'GET' - /api/v2/admin/channel-pricings/{id}` → `'GET' - /api/v2/admin/product-variants/{code}/pricing/{id}`
+
+* Order
+
+- `'GET' - /api/v2/admin/order-items/{id}` → `'GET' - /api/v2/admin/orders/{orderToken}/items/{id}`
+
+* Payment
+
+- `'GET' - /api/v2/shop/payments/{id}` → `'GET' - /api/v2/shop/orders/{orderToken}/payments/{id}`
+
+* Product
+
+- `'GET' - /api/v2/admin/product-images/{id}` → `'GET' - /api/v2/admin/products/{code}/images/{id}`
+- `'GET' - /api/v2/admin/products-images` → `'GET' - /api/v2/admin/products/{code}/images`
+- `'GET' - /api/v2/shop/product-images/{id}` → `'GET' - /api/v2/shop/products/{code}/images/{id}`
+- `'GET' - /api/v2/admin/product-option-values/{code}` → `'GET' - /api/v2/admin/product-options/{optionCode}/values/{code}`
+- `'GET' - /api/v2/shop/product-option-values/{code}` → `'GET' - /api/v2/shop/product-options/{optionCode}/values/{code}`
+
+* Promotion
+
+- `'GET' - /api/v2/admin/promotion-actions/{id}` → `'GET' - /api/v2/admin/promotions/{code}/actions/{id}`
+- `'GET' - /api/v2/admin/promotion-coupons/{id}` → `'GET' - /api/v2/admin/promotions/{code}/coupons/{id}`
```

Comment:
Not only GET has changed, but also POST (for creating and generating), PUT and DELETE